tests/test_aggregates_by_committee_id.py

```diff
@@ -0,0 +1,610 @@
+            candidate_id=self.candidate_17_only.candidate_id,
+            committee_id=self.committees_17_only[0].committee_id,
+            committee_designation='P',
+            committee_type='S',
+            cand_election_year=2017,
+            fec_election_year=2018,
+        )
+
+        # Create data for future presidential - next_cycle. Use formula for future
+
+        # Test full next_cycle and current_cycle 2-year totals
+
+        self.candidate_20 = factories.CandidateHistoryFutureFactory(
+            candidate_id='P456',
+            two_year_period=self.current_cycle,
+            candidate_election_year=self.next_cycle,
+        )
+        self.candidate_20 = factories.CandidateHistoryFutureFactory(
+            candidate_id='P456',
+            two_year_period=self.next_cycle,
+            candidate_election_year=self.next_cycle,
+        )
+        #Candidate history won't have next_cycle yet
+        self.committees_20 = [
+            factories.CommitteeHistoryFactory(cycle=self.current_cycle, designation='P'),
+        ]
+        factories.CandidateDetailFactory(
+            candidate_id=self.candidate_20.candidate_id,
+            election_years=[self.next_cycle],
+        )
+        [
+            factories.CandidateElectionFactory(
+                candidate_id=self.candidate_20.candidate_id,
+                cand_election_year=election_year
+            )
+            for election_year in [self.next_cycle - 4, self.next_cycle]
+        ]
+        [
+            factories.CommitteeDetailFactory(committee_id=each.committee_id)
+            for each in self.committees_20
+        ]
+        #Full next_cycle
+        factories.CandidateTotalFactory(
+            candidate_id=self.candidate_20.candidate_id,
+            cycle=self.next_cycle,
+            is_election=True,
+            receipts=55000,
+        )
+        #current_cycle 2-year
+        factories.CandidateTotalFactory(
+            candidate_id=self.candidate_20.candidate_id,
+            cycle=self.current_cycle,
+            is_election=False,
+            receipts=25000,
+        )
+        factories.CandidateFlagsFactory(
+            candidate_id=self.candidate_20.candidate_id
+        )
+        db.session.flush()
+
+        factories.CandidateCommitteeLinkFactory(
+            candidate_id=self.candidate_20.candidate_id,
+            committee_id=self.committees_20[0].committee_id,
+            committee_designation='P',
+            committee_type='P',
+            cand_election_year=self.next_cycle,
+            fec_election_year=self.current_cycle,
+        )
+
+        factories.CandidateCommitteeLinkFactory(
+            candidate_id=self.candidate_20.candidate_id,
+            committee_id=self.committees_20[0].committee_id,
+            committee_designation='P',
+            committee_type='P',
+            cand_election_year=self.next_cycle,
+            fec_election_year=self.next_cycle,
+        )
+
+    def test_by_size(self):
```

Comment:
Great work, @fec-jli! Did you mean to add tests for `test_by_size` and `test_by_state`? I only see data factories.

Reply:
Thank you for your review. 
because we are going to remove the 9 deprecated endpoint code later, this test tests/test_aggregates_by_committee_id.py will be removed together.
so I didn't spend more time to add more test on this.

I mean to add more test case on tests/test_aggregates.py , such as: test /schedules/schedule_a/by_zip/

Reply:
@fec-jli since these tests are for the deprecated endpoints, I'm ok moving on without the extra coverage. Thanks!